src/swagger-specs/catalog.yaml

```diff
@@ -2203,17 +2279,7 @@ definitions:
         description: >-
           ID of the last batch that modified the data set (only modified by Siphon batch ingest).
```

Comment:
This one can probably be turned into a full sentence, just a "The" in the beginning.